src/main/datatypes/MQ2BasicTypes.cpp

```diff
@@ -231,7 +231,7 @@ bool MQ2Int64Type::GetMember(MQVarPtr VarPtr, const char* Member, char* Index, M
 
 	case Int64Members::Prettify:
 		sprintf_s(DataTypeTemp, "%lld", VarPtr.Int64);
-		PrettifyNumber(DataTypeTemp, sizeof(DataTypeTemp), IsNumber(Index) ? atoi(Index) : 0);
```

Comment:
Do we have a solution for any existing code that might still be doing sizeof like here? Would rather it turn into some kind of compiler failure rather than misreporting the size

Reply:
I couldn't find a way to do it without using a script to check and providing a warning.  But the majority of usages used MAX_STRING instead of sizeof(DataTypeTemp) and the MAX_STRING usage is backwards compatible.

I searched RG, MMO, all plugins I have access to, and asked about usages from dewey, Sic, CWTN, and hytiek before I decided that the impact of this change is low (no usages of sizeof(DataTypeTemp) outside of core, and only a few in core).